app/database.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+engine = create_engine('sqlite:///data/examples/molssi_jobstore.db',
+                       convert_unicode=True)
```

Comment:
Move URI to the `config.py`
and read it here from the flask config as follows :
```python
from flask import current_app

uri = current_app.config['DATABASE_URI']
```

Remove any reference to db paths that is not defined in the config.py